Sources/SwiftInspectorVisitors/Tests/NestableTypeVisitorSpec.swift

```diff
@@ -233,6 +265,22 @@ final class NestableTypeVisitorSpec: QuickSpec {
             }
           }
 
+          context("with a single generic") {
```

Comment:
Do we have any tests for multiple generic parameters? Could be nice to have at least one.

Reply:
Good point! 04baaa1